x-pack/plugins/viz_editor/public/pseudo_plugins/scatter_chart_plugin/scatter_chart_plugin.tsx

```diff
@@ -220,15 +223,29 @@ function getSuggestionsForField(
     return [];
   }
 
+  const { datasource } = visModel;
+
+  const select: ColumnOperation[] = [
+    fieldToOperation(field, 'column') as ColumnOperation,
+    fieldToOperation(field, 'column') as ColumnOperation,
```

Comment:
Prompt to discussion: I just defaulted to same column on x and y because it's the easiest thing to do, should we maybe try to be smarter here and choose another number field from the datasource as y (and if there is more than one, just suggest them all?)

Reply:
Or we could switch to the scatter chart with only one axis chosen, and present the user with some kind of "Choose another axis" message? But I really like the idea of the user being able to see every possibility up front.